sci-physics/yoda/yoda-1.9.5-r1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit bash-completion-r1 autotools python-single-r1
+
+DESCRIPTION="Yet more Objects for (High Energy Physics) Data Analysis"
+HOMEPAGE="https://yoda.hepforge.org/"
+SRC_URI="https://yoda.hepforge.org/downloads?f=${P^^}.tar.bz2 -> ${P^^}.tar.bz2"
+S="${WORKDIR}/${P^^}"
+
+LICENSE="GPL-3"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="root python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} ) root? ( python )"
```

Comment:
Is this requirement correct? We need python for root? The previous versions had the root flag which worked without the python support.

Reply:
I think so because in the configure it states that `root` enables PyROOT interface: https://gitlab.com/hepcedar/yoda/-/blob/release-1-9-x/configure.ac#L66

The main reason for `yoda[root]` is to get the python scripts yoda2root and root2yoda from bin/, which are only available with `yoda[python]`.

Reply:
FYI: https://root.cern/manual/python/ PyROOT is a way to create automatic Python bindings for C++ code.